lib/rqliteDialect.ts

```diff
@@ -107,6 +105,10 @@ export class RqliteDialect extends Client implements Client_ {
     return defaults;
   }
 
+  wrapIdentifierImpl(value) {
+    return value !== "*" ? `\`${value.replace(/`/g, "``")}\`` : "*";
```

Comment:
I guess you got this little code piece from https://github.com/knex/knex/blob/29ba203/lib/dialects/sqlite3/index.js#L69? :)

Quick question: Same as with formatQuery, is there an advantage for copying the code, compared to using wrapIdentifierImpl from the sqlite3 dialect directly?

Example how I imagined this could work without copying the code:
```js
import Client_SQLite3 from "knex/lib/dialects/sqlite3";

...
...

RqliteDialect.prototype.wrapIdentifierImpl = Client_SQLite3.prototype.wrapIdentifierImpl
```